pandas/io/stata.py

```diff
@@ -3454,6 +3569,7 @@ def __init__(
         time_stamp: datetime.datetime | None = None,
         data_label: str | None = None,
         variable_labels: dict[Hashable, str] | None = None,
+        value_labels: dict[str, dict[float | int, str]] | None = None,
```

Comment:
Isn't this `dict[Hashable, ...]` since column names don't have to be strings (IIRC non strings are converted/munged until they satisfy Stata reqs)